packages/core/src/ssmDocument/ssm/ssmClient.ts

```diff
@@ -102,7 +102,7 @@ export async function activate(extensionContext: ExtensionContext) {
     // Create the language client and start the client.
     const client = new LanguageClient(
         'ssm',
-        localize('ssm.server.name', 'SSM Document Language Server'),
+        localize('ssm.server.name', 'AWS: SSM Document Language Server'),
```

Comment:
Use a common prefix for all Toolkit channels...

<img width="276" alt="image" src="https://github.com/aws/aws-toolkit-vscode/assets/55561878/298b4186-592a-4dff-b6d1-264b9a1bb664">